docs/source/matchcode/matchcode-pipeline.rst

```diff
@@ -4,11 +4,12 @@ Code Matching
 The aim of this tutorial is to show how to use the MatchCode.io API to perform
 code matching on an archive of files.
 
+
 .. note::
     This tutorial assumes that you have a working installation of PurlDB. If you
     don't, please refer to the `installation
-    <../purldb/overview.html#installation>`_ page.
-
+    <https://aboutcode.readthedocs.io/projects/PURLdb/en/latest/how-to-guides/installation.html>`_
```

Comment:
This should not be an absolute link to the readthedocs page. We want to use cross-references (https://www.sphinx-doc.org/en/master/usage/referencing.html)